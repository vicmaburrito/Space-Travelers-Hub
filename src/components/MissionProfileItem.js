import React from 'react';
import './MyProfile/MyProfile.css';
import PropTypes from 'prop-types';

function MissionProfileItem({ name }) {
  return (
    <div>
      <li className="border-bottom padLeftBottom">{name}</li>
    </div>
  );
}

export default MissionProfileItem;

MissionProfileItem.propTypes = {
  name: PropTypes.string,
};

MissionProfileItem.defaultProps = {
  name: undefined,
};
