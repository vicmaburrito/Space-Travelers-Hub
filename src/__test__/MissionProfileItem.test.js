import React from 'react';
import renderer from 'react-test-renderer';
import MissionProfileItem from '../components/MissionProfileItem';

it('renders correctly', () => {
  const tree = renderer.create(<MissionProfileItem />).toJSON();
  expect(tree).toMatchSnapshot();
});
