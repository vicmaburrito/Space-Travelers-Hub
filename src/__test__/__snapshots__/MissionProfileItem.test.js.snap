// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <li
    className="border-bottom padLeftBottom"
  />
</div>
`;
